docs/design/job_architecture.md

```diff
@@ -583,55 +550,36 @@ In kernel, as retrieved from EE2.check_job
     "retry_parent": null
 }
 ```
-#### BE Output State
+#### Job output state
 As sent to browser, includes cell info and run info
-```
-{
-    outputWidgetInfo: (if not finished, None, else...) job.get_viewer_params result
-    jobState: {
-        job_id: string,
-        status: string,
-        created: epoch ms,
-        updated: epoch ms,
-        queued: optional - epoch ms,
-        finished: optional - epoch ms,
-        terminated_code: optional - int,
-        tag: string (release, beta, dev),
-        parent_job_id: optional - string or null,
-        run_id: string,
-        cell_id: string,
-        errormsg: optional - string,
-        error (optional): {
-          code: int,
-          name: string,
-          message: string (should be for the user to read),
-          error: string, (likely a stacktrace)
-        },
-        error_code: optional - int
-        created: 1583863267000,
-        batch_id: str,
-        batch_job: bool,
-        child_jobs: array,
-        retry_ids: array,
-        retry_parent: str
-    }
-}
-```
-
-When an error occurs while preparing the job state, the output states will have the formats
 ```json
 {

```

Comment:
Should this structure use the abstracted names vs concrete names?

Reply:
I don't know.

Reply:
I would use one or the other consistently throughout the entire document. Since the abstracted names shouldn't change, theoretically, I'd probably go with those.

Reply:
I've put a note to indicate that the structure shows the type of the data in each field.

Reply:
I was asking whether the structure should be using the controlled vocab or not (currently not)